Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -365,7 +364,9 @@ ProcessCode TrackFindingAlgorithm::execute(const AlgorithmContext& ctx) const {
   TrackContainer tracksTemp(trackContainerTemp, trackStateContainerTemp);
 
   tracks.addColumn<unsigned int>("trackGroup");
+  tracks.addColumn<Acts::CombinatorialKalmanFilterTipState>("CkfTipState");
```

Comment:
I don't understand why this has to be done in `Examples`, rather than in Core. There is even a check at L1302 that this is defined, so why not define it by default, or in a Core factory?

If we have to define it here, shouldn't we use `Acts::s_combinatorialKalmanFilterTipStateColumn`?

Reply:
Maybe this should be put into a `CKF` function - not sure. The main reason I made it explicit on the caller side is that they have to care about this column later. If you copy containers you have to ensure the columns exist before copy is called. We also discussed this briefly @paulgessinger - any opinion?

Reply:
@timadye and I agreed on creating a factory method in the CKF which sets the track container up

Reply:
I implemented this now but did not utilize it in the Examples because we abstract the track finding there further

Reply:
Looks good! I commented on the Examples in-place, so I think we can close this thread.